emm-docs/content/subcommands/git-status.md

```diff
@@ -0,0 +1,20 @@
+---
+weight: 10 
+---
+The `git-status` subcommand will display the current status of the base repository and the 
+repository of all modules.
+
+```bash
+$ evg-module-manager git-status
+Status of enterprise:
+  On branch dbradf/my-test-branch
+  Changes to be committed:
+    (use "git restore --staged <file>..." to unstage)
+        modified:   README
+
+Status of base:
+  On branch dbradf/my-test-branch
+  Changes to be committed:
+    (use "git restore --staged <file>..." to unstage)
```

Comment:
This probably isn't possible but it might be nice to remove this line from the output as it might confuse users as to how to deal with these staged files.

Reply:
It is possible to filter these out, but I'm not sure we would want to. They provide useful clues as to what actions a user can perform, even if they are described in terms of git commands, they do map to commands that the tool supports.